remappings.txt

```diff
@@ -0,0 +1,6 @@
+ds-test/=lib/forge-std/lib/ds-test/src/
```

Comment:
why adding this file? is it part of Foundry framework?

Reply:
forge remappings > remappings.txt . I used this command after installing OpenZeppelin .